api_docs/markdowns/api/collections/collections.md

```diff
@@ -0,0 +1,227 @@
+# Collections API -- Getting Started [Alpha Version]
+
+## Summary
+
+The goal of this API is to give the possibility to cross data from different sources and different periods on a project. These data sources are named collections in this context and represent data coming from Botify Crawls, the Log Analyzer, Google Search Console and others.
+
+In the manner of an analytics tool, the collections API is queried by providing a set of dimensions and metrics to aggregate data.
+
+## Data Querying Endpoint
+
+- Operation: [[projectQuery;reference#/Project/projectQuery]]
+- Path: `projects/{username}/{project_slug}/query`
+- HTTP Verb: POST
+- Response: `Pagination<BQL_COLLECTIONS_RESULT>`
+
+Body:
+```JSON
+{
+  "collections": [COLLECTIONS],
+  "periods": [PERIODS],
+  "query": BQL_COLLECTIONS_QUERY
+}
+```
+
+### Collections
+
+A Botify Project has multiple collections usable for querying. To retrieve the available collections one can query the endpoint:
+
+- Operation: [[getProjectCollections;reference#/Collections/getProjectCollections]]
+- Path: `projects/{username}/{project_slug}/collections`
+- HTTP Verb: GET
+- Response: `List<Collection>`
+
+Three types of collections are to be distinguished:
+- Timestamped collections. Most collections are timestamped and represent a data source that evolves daily. This includes, among others, the Log Analyzer, Search Console and Visits features.
+- Non-timestamped collections. These collections represent a snapshot at a certain moment in time. This includes the Botify Crawl Reports.
+- "Global" collection. This unique collection is present on every project. It exposes dimensions that are available on all collections. This contains the URL Scheme and Segment data.
+
+The most frequently used collections are:
+- `crawl.YYYYMMDD`
+- `search_console`
+- `log_analyzer.google`
+
+### Collection details
+
+To get the available dimensions and metrics, a collection exposes the endpoint:
+- Operation: [[getCollectionDetail;reference#/Collections/getCollectionDetail]]
+- Path: `projects/{username}/{project_slug}/collections/{collection_id}`
+- HTTP Verb: GET
+- Response: `Datamodel`
+
+The `collection_id` is being retrieved through the collections list endpoint. It is also the name used for the collection on the data querying endpoint.
+For each field in the datamodel, a `kind` attributes specifies if a field is a dimension or a metric.
+
+[comment]: # (TODO: somehow provide a list of available dimensions and metrics. Maybe not exhaustive, so not necessarily dynamic)
```

Comment:
Do we leave the "comment" when merging? Probably not?

Reply:
The comment is not displayed on the rendered markdown, so I would suggest to keep it